src/app/pages/apps/components/installed-apps/app-row/app-row.component.html

```diff
@@ -7,13 +7,26 @@
         ix-auto-type="checkbox"
         [ixTest]="application.name"
         [checked]="application.selected"
+        (click)="$event.stopPropagation()"
         (change)="toggleAppChecked($event.checked)"
       ></mat-checkbox>
+      <img [src]="application.chart_metadata.icon" [src-fallback]="imagePlaceholder" />
```

Comment:
There is [app-card-logo component](https://github.com/truenas/webui/blob/master/src/app/pages/apps/components/app-card-logo/app-card-logo.component.ts#L13), please consider if it is applicable here.